packages/cli/src/LoadNodesAndCredentials.ts

```diff
@@ -125,19 +126,23 @@ class LoadNodesAndCredentialsClass {
 		const nodePackages = [];
 		try {
 			// Read downloaded nodes and credentials
-			const downloadedNodesFolder = UserSettings.getUserN8nFolderDowloadedNodesPath();
+			const downloadedNodesFolder = UserSettings.getUserN8nFolderDownloadedNodesPath();
 			const downloadedNodesFolderModules = path.join(downloadedNodesFolder, 'node_modules');
 			await fsAccess(downloadedNodesFolderModules);
 			const downloadedPackages = await this.getN8nNodePackages(downloadedNodesFolderModules);
 			nodePackages.push(...downloadedPackages);
-			// eslint-disable-next-line no-empty
-		} catch (error) {}
+		} catch (error) {
+			// Folder does not exist so ignore and return
+			return;
```

Comment:
In the original, if one of the community packages fails to load, we ignore the error and still load all previously successfully loaded packages. In this version, having a `return` in the catch block will prevent all packages from being loaded, even when some of them were successfully loaded. Is this what we want?

Reply:
Why are we not reporting the error here?

Reply:
if the code reaches this `catch` block, `nodePackages` is still an empty array, and the rest of the lines below this become dead code.

Reply:
You're right, I thought this runs inside a loop, but that's `loadDataFromPackage`.